packages/node_modules/action-chain/src/ActionBase.ts

```diff
@@ -6,22 +6,20 @@ export class StopExecution {
     this.value = value
   }
 }
+
+interface I<Context, InitialValue, Value = InitialValue>
+  extends ActionBase<Context, InitialValue, Value> {
+  (value: string): string
+  displayName: string
+}
+
 export class ActionBase<Context, InitialValue, Value = InitialValue> {
```

Comment:
Value generic isn't used elsewhere. Do we need it?

Reply:
tried in another branch
https://github.com/cerebral/overmind/compare/ac-ref2
should I replace PR with this variant?

Reply:
Yeah, this should be cleaned up :) I am working on a PR now which replaces `Context` and `Providers` with the term `Effects`. This gives us two things: